justfile

```
@@ -0,0 +1,46 @@
+set dotenv-load := false
+
+DOCKER_FILE := "-f docker-compose.yml"
+
+
+install:
+    #! /usr/bin/env sh
+    cd openverse-api
+    pipenv install --dev
+    pipenv run pre-commit install
+
+
+lint:
+    #! /usr/bin/env sh
+    cd openverse-api
+    pipenv run pre-commit run --all-files
+
+
+init: up
+    ./load_sample_data.sh
+
+
+healthcheck:
+    curl localhost:8000/v1/images/?q=honey
```

The unquoted URL for cURL causes issues with shells like `zsh`. Not when running with `just` but when running as a command directly. 

I'd recommend an optional small change.
```suggestion
    curl "localhost:8000/v1/images/?q=honey"
```